includes/class-wc-siftscience-order-status.php

```diff
@@ -0,0 +1,45 @@
+<?php
+/**
+ * This class handles the display of Sift feedback icons in order list and order view.
+ *
+ * @author Nabeel Sulieman, Rami Jamleh
+ * @package sift-for-woocommerce
+ * @license GPL2
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+if ( ! class_exists( 'WC_SiftScience_Order_Status' ) ) :
+
+	/**
+	 * Class WC_SiftScience_Orders
+	 */
+	class WC_SiftScience_Order_Status {
+
+		/**
+		 * Cache of order status options to avoid fetching it too many times
+		 *
+		 * @var array List of status options
+		 */
+		private $statuses = null;
```

Comment:
null is not needed (it's default but ok)

Reply:
This is a habit I picked up from C#: Always explicitly set the variable before using it (never rely on the default). If this was set in the constructor, then I would have removed `= null`.

example1:

```
$var; // not initialized because if-else will always set it before it's used
if ($condition) {
   $var = 1;
} else {
   $var = 0;
}
echo $var; 
```

example2:

```
$var = 0; // initialized to guarantee an explicit value before usage
if ($condition) {
   $var = 1;
} 
echo $var; 
```